portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/handlers/FavoritesHandler.java

```diff
@@ -118,18 +118,18 @@ private void saveUserFavorites(String userId, FavoriteSites favorites, boolean r
 
     public static class FavoriteSites {
 
-        private Set<String> favoriteSiteIds;
+        private List<String> favoriteSiteIds;
```

Comment:
@hornersa I used a Set to avoid dealing with duplicates so I would use a LinkedHashSet vs a List

Reply:
I'll force push this revision soon.